lib/autofs_utils.pm

```diff
@@ -134,6 +134,11 @@ sub configure_service {
         test_conf_file_content => $test_conf_file_content,
         test_mount_dir         => $test_mount_dir);
 
+    if (script_run('test -f /etc/nsswitch.conf') && !script_run('rpm -q glibc')) {
+        record_soft_failure('boo#20200812 openQA test fails in autofs - glibc-devel installs /usr/include/netdb.h which wrongly defines _PATH_NSSWITCH_CONF="/etc/nsswitch.conf" (/etc/nsswitch.conf was moved to /usr/etc/nsswitch.conf)');
```

Comment:
I don't understand this - the bugref seems to be wrong and what impact would `glibc-devel` have here?

Reply:
> AFAICT this PR is mixing four independent changes?

Yeah, maybe I should split the changes in more commits.

Reply:
> I don't understand this - the bugref seems to be wrong and what impact would `glibc-devel` have here?

`glibc-devel` owns `/usr/include/netdb.h` which is used by autofs to pick up the nsswitch configuration file. `/usr/include/netdb.h` points to a file, which has been moved. As there is no solution yet and the discussion continues in the bugref, I have rather included it here.

Reply:
Does it actually parse a C header file during runtime? I don't see any dependency here.

I assume it's a compile time thing, so the test for `glibc-devel` is a red herring.

Reply:
> I assume it's a compile time thing, so the test for `glibc-devel` is a red herring.

Agreed, removed from the test case.

Reply:
> I don't understand this - the bugref seems to be wrong and what impact would `glibc-devel` have here?

The bug ref was unfortunately missed to be fixed; this bug ref is invalid